mts/protocol/mtpresponder.h

```diff
@@ -162,7 +162,7 @@ private Q_SLOTS:
     QHash<MTPOperationCode, MTPCommandHandler>      m_opCodeTable;      ///< Hash table storing command handler functions
     StorageFactory                                 *m_storageServer;    ///< Pointer to the object storage server
     MTPTransporter                                 *m_transporter;      ///< Pointer to the transport layer
-    DeviceInfo                                     *m_devInfoProvider;  ///< Pointer to the device info class
+    MtpDeviceInfo                                     *m_devInfoProvider;  ///< Pointer to the device info class
```

Comment:
Looks like direct search & replace affected the alignment. Just cosmetics, though.

Reply:
Should be fixed.